real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/messages/IMUCompassMessage.java

```diff
@@ -0,0 +1,40 @@
+package com.roboclub.robobuggy.messages;
+
+import java.util.Date;
+
+/**
```

Comment:
Sounds good.  Please delete this documentation structure from this file and match that of the other files in this commit.